### DISCUSSION TITLE
Targeting Powershell 7+ from .NET Framework 4.7.2

### DISCUSSION THREAD

**Discussion by merlinschumacher** [PowerShell/PowerShell#18640] [Q&A]
Hi,

I'm trying to execute PowerShell code in a PowerShell 7 environment in .NET Framework 4.7.2 and C#. I can't find anything about a way to get this to work. Does anyone have an idea or tip, if it's possible at all?

Best regards and thanks for your help.

Merlin

**Comment by ALIENQuake** [comment]
@merlinschumacher Hi, PowerShell 6+ is built on NET 5+, not Net Framework.

**Comment by merlinschumacher** [comment]
I'm aware of that. I've tried remoting, which is also broken for PS 7 (See [#14274](https://github.com/PowerShell/PowerShell/issues/14274)) . 
Could there be another way?

**Comment by ALIENQuake** [comment]
Now I'm confused, what's remoting (broken or not) has to do with PS 7 running on .NET Framework 4.7.2 and c# ?

**Comment by rhubarb-geek-nz** [comment]
I would suggest run PowerShell 7 in a child process started from a .NET Framework process.
